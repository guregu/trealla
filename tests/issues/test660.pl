:- initialization(main).

main :-
     member(X,[true,false]),
     ( member(X,[false,true,false]) *->
         write(here1)
     ; write(here2)
     ),
     write({X}), fail.
main :- nl.
