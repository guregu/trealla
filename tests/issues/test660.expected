here1{true}here1{false}here1{false}
